NewEssentials/Commands/Chat/CSuperBroadcast.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Localization;
+using NewEssentials.Chat;
+using OpenMod.Core.Commands;
+using SDG.Unturned;
+using UnityEngine;
+using Color = System.Drawing.Color;
+using Command = OpenMod.Core.Commands.Command;
+
+namespace NewEssentials.Commands.Chat
+{
+    public class CSuperBroadcast : Command
+    {
+
+        private readonly IBroadcastingService m_Broadcasting;
+        private readonly IConfiguration m_Configuration;
+        private readonly IStringLocalizer m_Localizer;
+        
+        public CSuperBroadcast(IServiceProvider serviceProvider, IBroadcastingService broadcasting, IConfiguration config, IStringLocalizer localizer) : base(serviceProvider)
+        {
+            m_Broadcasting = broadcasting;
+            m_Configuration = config;
+            m_Localizer = localizer;
+        }
+
+        protected override async Task OnExecuteAsync()
+        {
+            if (Context.Parameters.Length < 1 || Context.Parameters.Length > 2)
+                throw new CommandWrongUsageException("/superbroadcast \"your text here\" <time>");
```

Comment:
We could add a [CommandSyntax] parameter and pass Context, let OpenMod handle telling the user the correct syntax. Maybe it's localized? Not sure.

Reply:
I was just replacing it from what I found lol, sure why not

Reply:
Yeah, for consistency reasons OpenMod should handle syntax